src/interface.jl

```diff
@@ -162,6 +170,11 @@ function forward(cb::Composed{<:Bijector}, x)
     return res
 end
 
+function rand(flow::Composed, dims::Integer, shape::Integer=1)
```

Comment:
`rand` for the transformed distribution is a requirement of Normalised Flows VI. 

Wouldn't this in general work for any composition of bijections?

Reply:
When I said it wouldn't work, I was referring to the fact that `transform` is no longer supported.
But I agree, with the change below, it would work not just for any _composition_, but for any `Bijector` I believe.

But I think it makes more sense to make this more general, allowing one to attach _any_ `Distribution` to a `Bijector`, composed or not. The `MultivariateTransformed` is the approach I've taken for this, and it seems to work nicely. Your function would then simply be a `Composed` bijector attached to a `MvNormal`. 

With your approach you might run into an issue where a function dispatches on `d::Distribution`, but `PlanarFlow() isa Distribution` is `false`. I think what I'm proposing with the transformed distributions is more modular and will "just work" in a lot of packages that uses `Distributions.jl`. For example, if you look at my PR for `Bijectors.jl` you can see an example where because of this `MultivariateTransformed` I could simply plug a transformed distribution into another package I'd implemend ages ago, and it _just worked_ :)

Does this make sense?

Reply:
I agree we don't need rand in Bijectors.jl